IoTHubManager/Constants.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace IoTHubManager
+{
+    class Constants
+    {
+
+        public const string TEST = "Type";
+        public const string INTEGRATION_TEST = "IntegrationTest";
+
+
+        //Web Service access URIs 
+        public class Urls
+        {
+            //IOTHUB Manager
+            public const string IOTHUB_ADDRESS = "http://localhost:9002/v1";
+            public const string DEVICE_PATH = "/devices";
+            public const string JOBS_PATH = "/jobs/";
+
+            //Device Simulation
+            public const string DEVICE_SIMULATION_ADDRESS = "http://localhost:9003/v1";
+            public const string SIMULATION_PATH = "/simulations/1";
+        }
+
+
+        //JSON template file paths. Templates for Devices and Jobs.
+        public class Path
+        {
+            private const string DEVICES_DIR = "./resources/devices/";
+            private const string JOBS_DIR = "./resources/jobs/";
+
+
+            //Device Files (JSON object holding device descriptions.)
+            public const string
+                DEVICE_FILE_AUTO_GEN_AUTH = DEVICES_DIR + "Device_Template_Auto_Generated_Auth.json",
+                DEVICE_FILE_SYMMETRIC_AUTH = DEVICES_DIR + "Device_Template_Symmetric_Auth.json",
+                DEVICE_FILE_X509_AUTH = DEVICES_DIR + "Device_Template_X509_Auth.json";
+
+            //Job Files (JSON object holding job (tag/reconfigure) descriptions.)
+            public const string
+                TAGS_FILE = JOBS_DIR + "Tags.json",
+                REBOOT_METHOD_FILE = JOBS_DIR + "Method_Chiller_Reboot.json",
+                FIRMWAREUPDATE_METHOD_FILE = JOBS_DIR + "Method_Elevator_FrimwareUpdate.json",
+                RECONFIGURE_DEVICE_FILE = JOBS_DIR + "Reconfigure_Chiller_Update_Model.json",
+                SIMULATION_FILE = JOBS_DIR + "Simulation.json";
+
```

Comment:
nit: remove unnecessary blank lines (here and where you have 2 blank lines in a row in this file)